src/highdicom/ann/content.py

```diff
@@ -71,6 +94,14 @@ def unit(self) -> CodedConcept:
         """highdicom.sr.CodedConcept: coded unit"""
         return self.MeasurementUnitsCodeSequence[0]
 
+    @property
+    def referenced_images(self) -> Union[ReferencedImageSequence, None]:
+        """Union[highdicom.ReferencedImageSequence, None]: referenced images"""
+        if hasattr(self, 'ReferencedImageSequence'):
+            return self.ReferencedImageSequence
```

Comment:
Currently this will not return the type suggested by the type hint. I think you need this:

```suggestion
            return ReferencedImageSequence.from_sequence(self.ReferencedImageSequence)
```